tests/index.js

```diff
@@ -14,7 +14,7 @@ var mongouri = 'mongodb://127.0.0.1/mongoose-buv-'
 
 console.log('Connecting to ' + mongouri + '...');
 
-mongoose.connect(mongouri, {useMongoClient: true}).then(function () {
+mongoose.connect(mongouri).then(function () {
```

Comment:
This change triggers a warning on Mongoose < 5, which can be avoided if we pass the option conditionally based on the currently installed Mongoose version.